prep-javascript-conditionals/main.js

```diff
@@ -0,0 +1,70 @@
+/* Your JS goes in this file */
+function isAdult(age) {
+  if (age >= 18) {
+    return true;
+  }
+  if (age <= 18) {
```

Comment:
Close! 18 would be an adult. So in this case you would want to check if `age < 18` to confirm that they are not an adult.

Reply:
or `age <= 17`